docs/about.md

```diff
@@ -110,3 +110,8 @@ editor while it's running, adding the new features.
 ## Open Source
 
 MakeCode is an open source project and is provided as a joint effort between [Microsoft Research](https://www.microsoft.com/en-us/research/project/microsoft-makecode/) and [Visual Studio](https://visualstudio.microsoft.com/).
+
+
+## Contact us
+
+Trying to reach us? Try [the contact page](/contact).
```

Comment:
Trying sounds desperate. Maybe "Want to reach us? Get in touch using one of the resources on the [contact page](/contact)."

Reply:
Are you *dying* to talk to us???

Reply:
Well, if this is the killer app, then yes.